coalib/parsing/ConfParser.py

```diff
@@ -82,14 +83,15 @@ def __add_comment(self, section, comment, origin):
         section.append(Setting(
             key,
             comment,
-            origin,
+            SourcePosition(str(origin)),
```

Comment:
a SourcePosition is only useful if we are adding a line number.

Either add a line number here, or restore this to be only `origin`